js1/dom-innerhtml/dom-elementy.md

```diff
@@ -130,13 +117,13 @@ Vybírat prvky na stránce pomocí CSS tříd se občas velmi hodí. Jindy nás
 Funkce `document.querySelector` vždy vrátí **první** prvek, který na stránce najde. Pokud se chceme dostat například ke druhé kartě, pomocí selectoru `.card` to nepůjde. Takto získáme pouze kartu první.
 
 ```js
-const cardElm = document.querySelector('.card');
+const cardElement = document.querySelector('.card');
 ```
 
 Jelikož lze v `document.querySelector` použít libovolný CSS selektor, někoho by mohla napadnout například takováhle frajeřinka.
 
 ```js
-const cardElm = document.querySelector('.card:nth-child(2)');
+const cardElement = document.querySelector('.card:nth-child(2)');
 ```
 
 Tento kód bude skutečně fungovat, ale rovnou zde na místě vám takové triky navždy zakazuju. Takovéto komplikované selektory používejte pouze v případě, že opravdu není vyhnutí. Pro nás existuje jednodušší řešení a to dát každé kartě unikátní `id`.
```

Comment:
Tady navrhuju napsat "zakazujeme", ať je to konzistentní.

Reply:
Přetextoval jsem to trochu víc. Slovo zakazujeme mi přijde až moc agresivní a trochu se mi nehodí do krámu pro piškvorky, které na tom `nth` z části stojí.

Reply:
Souhlasím